### PR TITLE
Add smoke test to test monitoring/v3 using gapic python client library.

### PR DESCRIPTION
**Pull request by ethanbao**

### CODE REVIEW

File: test/monitoring/test.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+from google.cloud.gapic.monitoring.v3 import group_service_api
+from google.cloud.gapic.monitoring.v3 import metric_service_api
+from google.cloud.gapic.monitoring.v3 import enums
+from google.gax import CallOptions, INITIAL_PAGE
+from google.monitoring.v3 import group_pb2
+from google.monitoring.v3 import common_pb2
+from google.monitoring.v3 import metric_pb2
+from google.protobuf import timestamp_pb2
+from google.api import metric_pb2 as api_metric_pb2
+
```
Comment (bjwatson) [inline test/monitoring/test.py]: Double linebreak (that's the Pythonic way)

File: test/monitoring/test.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+PROJECT_ID = '<PROJECT_ID>'
+
```
Comment (bjwatson) [inline test/monitoring/test.py]: Double linebreak

File: test/monitoring/test.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    print(member)
+
+  print "Delete groups:\n"
+  name = api.group_path(PROJECT_ID, groupId)
+  api.delete_group(name)
+  try:
+    response = api.get_group(name)
+  except:
+    print "Expect error here since the group should have been deleted"
+
```
Comment (bjwatson) [inline test/monitoring/test.py]: Double linebreak

File: test/monitoring/test.py [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  name = api.project_path(PROJECT_ID)
+  # Iterate over all results
+  for metric in api.list_metric_descriptors(name):
+    # process element
+    print metric
+
+  print "delete metric:"
+  name = api.metric_descriptor_path_path(PROJECT_ID, custom_metric_type)
+  api.delete_metric_descriptor(name)
+
```
Comment (bjwatson) [inline test/monitoring/test.py]: Double linebreak

File: test/monitoring/test.py [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+  filter_ = (
+    'metric.type = "compute.googleapis.com/instance/cpu/usage_time" '
+    'AND metric.label.instance_name = "endpoints11-load-test"')
+
+  view = enums.ListTimeSeriesRequest.TimeSeriesView.FULL
+  for element in api.list_time_series(name, filter_, interval, view):
+      # process element
+      print element
+
```
Comment (bjwatson) [inline test/monitoring/test.py]: Double linebreak

File: test/monitoring/test.py [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+  name = api.monitored_resource_descriptor_path(PROJECT_ID, 'uptime_url')
+  response = api.get_monitored_resource_descriptor(name)
+  print "Get monitored resource\n%s" % response
+
+testGroupCollection()
+testTimeseriesCollection()
+testMonitoredResourceCollection()
+testMetricCollection()
+
```
Comment (bjwatson) [inline test/monitoring/test.py]: Remove linebreak

File: test/monitoring/test.py [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+  print "List monitored reourses:"
+  # Iterate over all results
+  for element in api.list_monitored_resource_descriptors(name):
+    # process element
+    print element
+
+  name = api.monitored_resource_descriptor_path(PROJECT_ID, 'uptime_url')
+  response = api.get_monitored_resource_descriptor(name)
+  print "Get monitored resource\n%s" % response
+
```
Comment (bjwatson) [inline test/monitoring/test.py]: Double linebreak

File: test/monitoring/test.py [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+  response = api.get_group(name)
+  print "Group fetched: \n%s\n" % response
+
+  print "List groups:\n"
+  for group in api.list_groups(name):
+      # process element
+      print(group)
+
+  print "List group members:\n"
+  name = api.group_path(PROJECT_ID, groupId)
```
Comment (bjwatson) [inline test/monitoring/test.py]: If you repeat this here, you should also repeat it for `list_groups`

File: test/monitoring/test.py [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+
+  metric_descriptor = api_metric_pb2.MetricDescriptor(
+    type=custom_metric_type,
+    metric_kind=enums.MetricDescriptor.MetricKind.GAUGE,
+    value_type=enums.MetricDescriptor.ValueType.BOOL)
+  response = api.create_metric_descriptor(name, metric_descriptor)
+  print "Metric created: \n%s\n" % response
+
+  # This part is flaky as the backend will sometimes return NOT_FOUND for the
+  # metric created in last step.
```
Comment (bjwatson) [inline test/monitoring/test.py]: Put a sleep in here, like @geigerj did in https://github.com/googleapis/api-client-staging/pull/78

Comment (ethanbao) [inline test/monitoring/test.py]: I tried sleep earlier, which didn't seem to solve the problem.

File: test/monitoring/test.py (hunk 1 above)
Comment (ethanbao) [inline test/monitoring/test.py]: Done

File: test/monitoring/test.py (hunk 8 above)
Comment (ethanbao) [inline test/monitoring/test.py]: Done

File: test/monitoring/test.py (hunk 3 above)
Comment (ethanbao) [inline test/monitoring/test.py]: Done

File: test/monitoring/test.py (hunk 9 above)
Comment (ethanbao) [inline test/monitoring/test.py]: Done

File: test/monitoring/test.py (hunk 4 above)
Comment (ethanbao) [inline test/monitoring/test.py]: Done

File: test/monitoring/test.py (hunk 5 above)
Comment (ethanbao) [inline test/monitoring/test.py]: Done

File: test/monitoring/test.py (hunk 7 above)
Comment (ethanbao) [inline test/monitoring/test.py]: Done

**Comment by ethanbao** [comment]
PTAL

**Pull request closed without merging**